stable/spinnaker/values.yaml

```diff
@@ -71,25 +51,9 @@ deck:
     #    hosts:
     #      - domain.com
 
-gate:
-  allowedOriginsPattern: '^https?://(?:localhost|127.0.0.1|[^/]+\.example\.com)(?::[1-9]\d*)?/?$'
-
 # Bucket to use when storing config data in S3 compatible storage
 storageBucket: spinnaker
```

Comment:
Would it be clearer to denormalize (for lack of a better term) `storageBucket` into the various storage providers' config blocks? Unless it gets used somewhere that I'm not seeing it seems like it should be configured alongside the storage provider, rather than outside of it as a top level key.